homeassistant/components/scene/services.yaml

```diff
@@ -54,3 +54,12 @@ create:
       selector:
         entity:
           multiple: true
+
+delete:
```

Comment:
I'm not sure why we have the service description for the platform specific services in the scene integration and not under the integration that registered the service. It may be some legacy thing that we haven't updated. Can you try moving this to under the `homeassistant` integration inside a `services.yaml` file?